platforms/chibios/sleep_led.c

```diff
@@ -190,7 +193,7 @@ void sleep_led_toggle(void) {
 void sleep_led_init(void) {}
 
 void sleep_led_enable(void) {
-    led_set(1 << USB_LED_CAPS_LOCK);
+    led_set(2); // Caps Lock
```

Comment:
Would it not be better to avoid magics by doing the same as whats above 
```
        led_state.caps_lock = true;
        led_set(led_state.raw);
```

Reply:
This is just a temporary measure; the plan is to get rid of everything in the indicator API that operates on uint8_t, and replace it with led_t. I just didn't want to lump all of that extra work in this PR.